src/missing_prices.py

```diff
@@ -112,35 +117,29 @@ def fetch_tokens_without_prices(dune: DuneAPI) -> list[CoinPaprikaToken]:
     return [CoinPaprikaToken.from_dict(r) for r in results]
 
 
-def run_missing_prices():
+def run_missing_prices() -> None:
     """Script's Main Entry Point"""
     print("Getting Coin Paprika token list")
     coins = load_coins()
     print(f"Loaded {len(coins)} coins from Coin Paprika")
 
-    # Fetch tokens orders by descending, popularity
-    print("Getting traded tokens without prices from: https://dune.com/queries/224239")
-    tokens = sorted(
-        # Could be interesting to just get the results without executing.
-        fetch_tokens_without_prices(dune=DuneAPI.new_from_environment()),
-        key=lambda t: t.popularity,
-        reverse=True,
-    )
+    tokens = load_tokens(DuneClient(api_key=os.environ["DUNE_API_KEY"]))
```

Comment:
What's your opinion on adding a `load_dotenv()` line here like you did for `missing_tokens.py` because otherwise it won't run for me.